doc/hands_on/hands_on_parquet.md

```diff
@@ -0,0 +1,240 @@
+          "decimalPrecision": 20,
+          "decimalScale": 0
+        }
+      },
+      "Price": {
+        "toVarcharMapping": {
+          "destinationName": "PRICE",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "nonStringBehaviour": "CONVERT_OR_ABORT",
+          "overflowBehaviour": "TRUNCATE"
+        }
+      },
+      "Title": {
+        "toVarcharMapping": {
+          "destinationName": "TITLE",
+          "key": "none",
+          "required": false,
+          "varcharColumnSize": 2000000,
+          "nonStringBehaviour": "CONVERT_OR_ABORT",
+          "overflowBehaviour": "TRUNCATE"
+        }
+      }
+    }
+  },
+  "$schema": "https://schemas.exasol.com/edml-1.3.0.json"
+}
+```
+
+The generator writes the mapping without formatting. For easier viewing and editing use an editor / IDE with good JSON support and format the file. Good editors also help with editing because they can give you code-completion based on the JSON-schema.
+
+## Improving the Mapping
+
+You can edit this mapping to fine tune how the virtual schema maps the data. For example, you can change the varchar column sizes, if you know that your data has a certain max-size (2,000,000 is the Exasol limit). For details on the EDML mapping language check the [edml user-guide][edml-guide].
+
+One thing that you should always check in the generated file is the `source` property. This property must point to the source file(s) in your S3 bucket. So if you just uploaded the `books.parquet` file to the bucket, the generated value is ok. If you renamed it or moved it to a subfolder you need to change the name in the mapping definition.
+
+You also need to change it here if your parquet data is split into parts. In that case change the source to something like `books_part_*.parquet`.
+
+You can also modify the order of the columns by simply reordering the mapping definitions in the mapping file.
+
+Now upload this mapping to BucketFS:
+
+``` shell script
+curl -I -X PUT -T mapping.json http://w:writepw@<YOUR_DB_IP>:2580/default/
+```
+
+## Creating the Virtual Schema
+
+Now we can create the Virtual Schema:
+
+```sql
+CREATE VIRTUAL SCHEMA FILES_VS_TEST USING ADAPTER.S3_FILES_ADAPTER WITH
+    CONNECTION_NAME = 'S3_CONNECTION'
+    SQL_DIALECT     = 'S3_DOCUMENT_FILES'
+    MAPPING         = '/bfsdefault/default/mapping.json';
+```
+
+## First Query
+
+It's time to run our first query!
+So let's get the title of all our books by running:
+
+```sql
+SELECT * FROM FILES_VS_TEST.BOOKS;
+```
+
+![Screenshot of the result of the ](./parquetQueryResult.png)
+
+## What's next
+
+So now you know everything for creating your first S3 Virtual Schema. If you want to do more advanced mappings , check out the [EDML documentation][edml-guide]. For example, you could change the mapping so that the nested list authors is no longer mapped to a column with JSON string but to a separate table.
```

Comment:
```suggestion
So now you know everything for creating your first S3 Virtual Schema. If you want to do more advanced mappings, check out the [EDML documentation][edml-guide]. For example, you could change the mapping so that the nested list authors are no longer mapped to a column with JSON string but to a separate table.
```